ssl/t1_lib.c

```diff
@@ -1771,7 +1771,7 @@ void tls1_get_formatlist(SSL_CONNECTION *s, const unsigned char **pformats,
     if (s->ext.ecpointformats) {
         *pformats = s->ext.ecpointformats;
         *num_formats = s->ext.ecpointformats_len;
-    } else if (s->options & SSL_OP_LEGACY_EC_POINT_FORMATS) {
+    } else if (s->options & SSL_OP_LEGACY_EC_POINT_FORMATS != 0) {
```

Comment:
Oh, sorry for incorrect suggestion. It should be `(s->options & SSL_OP_LEGACY_EC_POINT_FORMATS) != 0`

Reply:
Yep, now fixed, thanks. Classic example of being too confident in a tiny change to bother rebuilding locally :laughing: